### DISCUSSION TITLE
New font

### DISCUSSION THREAD

**Discussion by Zakichanu** [heyform/heyform#73] [Ideas]
Sorry, it is a tiny, but it is possible to add [RAJDHANI font](https://fonts.adobe.com/fonts/rajdhani) on the font selection ? Thanks a lot

**Comment by iMuFeng** [comment]
Added in [0a14f15](https://github.com/heyform/heyform/commit/0a14f15a2af6cc8de0908113852f27799e383984)

**Comment by Zakichanu** [comment]
A massive thanks to you !
